supabase/migrations/20231008201517_enum_transaction_types.sql

```diff
@@ -1,4 +1,4 @@
-create type transaction_type as enum ('balance', 'expense');
+create type transaction_type as enum ('income', 'expense');
```

Comment:
Got this @arnellebalane will create a new migration file or apply the necessary changes if we have a final enum values

Reply:
For me @arnellebalane @japhethbalane it would be better to call it income so we can reserve the word balance for the user balance API (income - expense). What are your thoughts?

Reply:
@PCalape @japhethbalane Not all money coming into the balance are income though, could be other things like allowance or others.

Reply:
Ohhhh that is right @arnellebalane, it looks like balance would be better. Will proceed to use balance